src/test/java/pl/team/carrent/repository/EmployeeRepositoryTest.java

```diff
@@ -71,7 +69,7 @@ public void shouldFindBySurname() {
         //given
         init();
```

Comment:
Jeśli init wykonujesz co test to możesz wykorzystać sobie adnotację `@Before` nad metodą `init()`, dzięki temu nie będziesz musiał wołać jej sam co test